modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductSKUs.testcase

```diff
@@ -254,6 +254,70 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-10604. Can patch a SKU's ERC via API"
+	@priority = 3
+	test CanPatchSKUERCViaAPI {
+		property portal.acceptance = "false";
+
+		var skuId = CommerceJSONProductsAPI._getSKUId(productSKU = "MIN93015");
+
+		task ("Given a Minium site is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("And a default ERC 'MIN93015minium-initializer' for 'MIN93015' product is set") {
+			CommerceProducts.openProductsAdmin();
+
+			CommerceEntry.gotoMenuTab(menuTab = "SKUs");
+
+			CommerceProducts.searchAndGoToProductEntry(entryProductName = "MIN93015");
+
+			SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+			AssertTextEquals(
+				locator1 = "CommerceEntry#SKU_ERC_FIELD",
+				value1 = "MIN93015minium-initializer");
+		}
+
+		task ("When both null and empty values are patched and passed via API, the ERC for product is not updated") {
```

Comment:
After the "When" task, for the assertion, you can divide it into two tasks, first When task, second Then task for assertion. 

The same thing also in the And task that follows, first a When task, for assertion a Then task could be more clear.